lib/hanami/slice_registrar.rb

```diff
@@ -46,19 +46,16 @@ def freeze
     end
 
     def load_slices
-      slice_configs = Dir[root.join(CONFIG_DIR, SLICES_DIR, "*#{RB_EXT}")]
-        .map { |file| File.basename(file, RB_EXT) }
+      slice_configs = root.join(CONFIG_DIR, SLICES_DIR).glob("*#{RB_EXT}")
+        .map { _1.basename(RB_EXT) }
```

Comment:
This is a good use of `_1` IMO, especially when combined with `Pathname#basename`. Appreciate you.

Reply:
(General appreciation extends to the similar refactoring in the stanza below)